index.bs

```diff
@@ -695,7 +695,60 @@ For instance, an {{MLOperand}} may represent a constant feeding to an operation
 interface MLOperand {};
 </script>
 
-See also [[#security-new-ops]]
+{{MLOperand}} has the following internal slots:
+<div class=internal-slots>
+<dl dfn-type=attribute dfn-for="MLOperand">
+    : <dfn>\[[builder]]</dfn> of type {{MLGraphBuilder}}
+    ::
+        The {{MLOperand}}'s associated builder object.
+
+    : <dfn>\[[kind]]</dfn> of type {{string}}
+    ::
+        The {{MLOperand}}'s kind, i.e. "input", "constant" or "output".
```

Comment:
I'm still a bit confused with how the MLOperand kind is determined (e.g. "input" vs. "output"). For example, when an operand is created, what is the internal value set to the kind slot? Then when the same operand is immediately passed into the construction of the next operand in the ordering of the graph, when and what value should now be set on the kind slot? I'm still questioning if we need it, even as an internal slot. What value do we want to get out of defining it in the implementation of `MLOperand`?

Reply:
I am not against testing if we need it at all, but the implementation certainly uses the "kind". 

As explained above, at least  the "constant" vs "input" differentiation is need in the build steps (currently missing).

I see no ham in keeping this slot, or even the IDL definitions, until we get the algorithms (at least the build algorithm) sorted out. Of course we can remove anything that is not strictly needed.

@huningxin what is your take on this?

Reply:
The platform operand object ([[operand]] internal slot) should already take the kind info, e.g. "constant", when it is being created. The algorithm step is

> Make a request to the underlying platform to register |operand| as a tensor constant with |bytes| as value and store a reference to the corresponding [=implementation-defined=] object to |operand|.{{MLOperand/[[operand]]}}.

So the internal slot [[kind]] probably can be removed unless it is useful for any other algorithm steps.

Reply:
Currently, in the scope of this PR the [[kind]] slot is not needed.
It was added for future dependency, as it turns out, mainly for the build steps.

I guess the simplest is to remove it from the scope of this PR, and eventually re-add later when the build steps PR is filed. Then we can discuss there if/how we can do without it.